src/parser.js

```diff
@@ -77,7 +77,18 @@ function parseWithBabylon(text) {
       "functionSent",
       "dynamicImport"
     ]
-  });
+  };
+
+  try {
+    return babylon.parse(text, babylonOptions);
+  } catch (originalError) {
+    babylonOptions.strictMode = false;
```

Comment:
please don't mutate options, if we were going to use it another time, it would not work and likely to cause bugs. Instead do

```js
return babylon.parse(text, Object.assign({}, babylonOptions, {strictMode: false}));
```